MakeTools/funcs-ext.sh

```diff
@@ -825,6 +831,9 @@ function os_vendor_version()
 	mandrake)
 		rval=`cat /etc/mandrake-release | cut -d' ' -f4`
 		;;
+	opencloudos)
+		rval=`cat /etc/opencloudos-release | cut -d' ' -f4`
```

Comment:
I downloaded the 8.6 rpm and `/etc/opencloudos-release` appears to only have 2 fields. so I think this should be `| cut -d' ' -f2`

Reply:
I forgot to mention that we are working on a stream release which is different from 8.6.
https://mirrors.opencloudos.tech/opencloudos-stream/updates/2301/x86_64/Packages/opencloudos-stream-release-2301-6.ocs1.noarch.rpm

cat /etc/opencloudos-release
OpenCloudOS Stream release 2301

Reply:
opencloudos 8.6 is based on centos, but the stream release is not.

Reply:
this file is a symlink to the stream file. I think you should read from the stream file `opencloudos-stream-release` as the formats seem different.

Reply:
ok, fixed it.